src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,35 @@
+import { Modal as ChakraModal } from "@chakra-ui/react";
```

Comment:
No reason to change it as the work is already done. But for future reference, there is a more concise syntax in which the components are imported as part of the same command. 

See [Table](https://github.com/NYCPlanning/ae-streetscape/blob/main/src/components/Table/Table.tsx) or [Transitions](https://github.com/NYCPlanning/ae-streetscape/blob/main/src/components/Transitions/Transitions.tsx) for an example.

(I also understand that we might want to keep the syntax here because it's more readable to pair each Component import with its Props import)